frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -67,7 +67,7 @@ function ReaderPaging:registerKeyEvents()
         self.key_events.GotoNextPos = { { "Down" }, event = "GotoPosRel", args = 1, }
         self.key_events.GotoPrevPos = { { "Up" }, event = "GotoPosRel", args = -1, }
     end
-    if Device:hasKeyboard() then
+    if Device:hasKeyboard() and not Device.alt_plus_key_kernel_translated then
         self.key_events.GotoFirst = { { "1" }, event = "GotoPercent", args = 0,   }
         self.key_events.Goto11    = { { "2" }, event = "GotoPercent", args = 11,  }
```

Comment:
These ones read a bit not obvious: it's not because a Device has `alt_plus_key_kernel_translated` that 1, 2, 3... would not work (some Kindle77 may translate XWCVB to $*!/?.
So, may be keep `Device.model ~= "Kindle3"` for these ?